spec/lib/annotate/annotate_models_spec.rb

```diff
@@ -3048,6 +3049,58 @@ class User < ActiveRecord::Base
       end
     end
 
+    describe 'if a table has a commented column' do
+      it 'should be added the column with comment' do
+        annotation_options = { with_comment: true, position: :before }
+        klass = mock_class(:'users',
+                           :id,
+                           [
+                             mock_column(:id, :integer),
+                             mock_column(:name, :string, comment: 'commented column')
+                           ])
+        schema_info = AnnotateModels.get_schema_info(klass, '== Schema Info', annotation_options)
+        AnnotateModels.annotate_one_file(@model_file_name, schema_info, :position_in_class, annotation_options)
+        expect(File.read(@model_file_name)).to include('commented column')
+      end
+
+      context 'and the model already has annotation and the commented column is updated to non-null' do
+        before do
+          @annotation_options = { with_comment: true, position: :before }
+          klass = mock_class(:'users',
+                             :id,
+                             [
+                               mock_column(:id, :integer),
+                               mock_column(:name, :string, comment: 'commented column', null: true)
+                             ])
+          schema_info = AnnotateModels.get_schema_info(klass, '== Schema Info', @annotation_options)
+          AnnotateModels.annotate_one_file(@model_file_name, schema_info, :position_in_class, @annotation_options)
+        end
+
+        it 'should be added non-null option in the commented column' do
+          klass = mock_class(:'users',
+                             :id,
+                             [
+                               mock_column(:id, :integer),
+                               mock_column(:name, :string, comment: 'commented column', null: false)
+                             ])
+          schema_info = AnnotateModels.get_schema_info(klass, '== Schema Info', @annotation_options)
+          expect do
+            AnnotateModels.annotate_one_file(
+              @model_file_name,
+              schema_info,
+              :position_in_class,
+              @annotation_options
+            )
+          end
+            .to(
+              change { File.read(@model_file_name) }
+                .from(include("name(commented column) :string\n"))
+                .to(include("name(commented column) :string           not null\n"))
+            )
```

Comment:
If parentheses aren't added, this annotation isn't updated and this test is failed with the following error.

```
  1) AnnotateModels annotating a file if a table has a commented column and the model already has annotation and the commented column is updated to non-null should be added non-null option in the commented column
     Failure/Error:
       expect do
         AnnotateModels.annotate_one_file(@model_file_name, schema_info, :position_in_class, @annotation_options)
       end.to change { File.read(@model_file_name) }
                .from(include("name(commented column) :string\n"))
                .to(include("name(commented column) :string           not null\n"))

       expected `File.read(@model_file_name)` to have changed from include "name(commented column) :string\n" to include "name(commented column) :string           not null\n", but did not change
     # ./spec/lib/annotate/annotate_models_spec.rb:3087:in `block (5 levels) in <top (required)>'
```